carbon/job.py

```diff
@@ -46,6 +47,14 @@ def __init__(self, job_type: str) -> None:
         self.job_type = job_type
 
 
+class JobState(Enum):
+    """Enumeration of supported job states."""
+
+    FINISHED = "F"
+    RUNNING = "R"
+    EXPIRED = "X"
```

Comment:
There are several places below where you use `["F", "R", "X"]` but it would be better to use the values of the enum here.